temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
```

Comment:
Import the `json` module at the beginning of the file to ensure that `json.dumps` can be used within the `reset_files` function. [important]